crates/forge/tests/e2e/cores.rs

```diff
@@ -69,12 +71,14 @@ fn with_more_than_one_core() {
         [..]Finished[..]
 
 
-        Collected 11 test(s) from simple_package package
-        Running 1 test(s) from src/
+        Collected 13 test(s) from simple_package package
+        Running 2 test(s) from src/
         [PASS] simple_package::test_fib
-        Running 10 test(s) from tests/
+        [IGNORE] simple_package::ignored_test
```

Comment:
We should just print [SKIP] we don't need special tag for it

Reply:
Discussed offline, leaving it for now

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/926